### PR TITLE
Implemented query RDS without Backup #1426

### LINKED ISSUE #1426: Add RDS without Backup query for Ansible
### Platform
Ansible

### Description
RDS configured without backup

### PR DESCRIPTION
**Pull request by samuel-ferreira**
Implemented query RDS without Backup

Closes #1426

### CODE REVIEW

File: assets/queries/ansible/aws/rds_without_backup/query.rego [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  instanceName := task.name
+
+  object.get(instance, "backup_retention_period", "undefined") == "undefined"
+
+  result := {
+                "documentId":       input.document[i].id,
+                "searchKey":        sprintf("name={{%s}}.{{community.aws.rds_instance}}", [instanceName]),
+                "issueType":        "MissingAttribute",
+                "keyExpectedValue": "community.aws.rds_instance should have the property 'backup_retention_period' greater than 0",
+                "keyActualValue":   "community.aws.rds_instance has the property 'password_reuse_prevent' unassigned"
```
Comment (ruigomescx) [inline assets/queries/ansible/aws/rds_without_backup/query.rego:17]: fix actual value to report missing attribute backup_retention_period

File: assets/queries/ansible/aws/rds_without_backup/query.rego [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  instanceName := task.name
+
+  instance.backup_retention_period == 0
+
+  result := {
+                "documentId":       input.document[i].id,
+                "searchKey":        sprintf("name={{%s}}.{{community.aws.rds_instance}}.backup_retention_period", [instanceName]),
+                "issueType":        "IncorrectValue",
+                "keyExpectedValue": "community.aws.rds_instance should have the property 'backup_retention_period' greater than 0",
+                "keyActualValue":   "community.aws.rds_instance has the property 'password_reuse_prevent' assigned to 0"
```
Comment (ruigomescx) [inline assets/queries/ansible/aws/rds_without_backup/query.rego:35]: fix actual value to report backup_retention_period field

**Review by ruigomescx — CHANGES_REQUESTED** [review]

File: assets/queries/ansible/aws/rds_without_backup/query.rego (hunk 1 above)
Comment (samuel-ferreira) [reply assets/queries/ansible/aws/rds_without_backup/query.rego]: Fixed

File: assets/queries/ansible/aws/rds_without_backup/query.rego (hunk 2 above)
Comment (samuel-ferreira) [reply assets/queries/ansible/aws/rds_without_backup/query.rego]: Fixed

**Pull request merged**